webrtc.html

```diff
@@ -5462,6 +5462,19 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
     connections between participants. One will normally assume that
     the IP address of TURN servers is not sensitive information.</p>
 
+    <p>Mitigating the exposure of IP addresses to the application requires
+    limiting the IP addresses that can be used, which will impact the
+    ability to communicate on the most direct path between endpoints.
+    Browsers are encouraged to provide appropriate controls for deciding
+    which IP addresses are made available to applications, based on the
+    security posture desired by the user.</p>
+
```

Comment:
Has the WG discussed and come to consensus around this because my understanding was we were still at the point that WebRTC would expose internal IP addresses by default. Perhaps I am just missing it but this is such a hot thing, I'd prefer if someone could point the editors at the discussion around where we decided this before we merge this. I'm not claiming I definitely know what the consensus is, I just think we need to go make check this text roughly matches the discussion before we put it in.